components/camel-twitter/src/test/java/org/apache/camel/component/twitter/CamelComponentVerifierTest.java

```diff
@@ -54,9 +55,9 @@ public void testInvalidKeyConfiguration() {
 
         Assert.assertEquals(ComponentVerifier.Result.Status.ERROR, result.getStatus());
         Assert.assertEquals(1, result.getErrors().size());
-        Assert.assertEquals(ComponentVerifier.CODE_AUTHENTICATION, result.getErrors().get(0).getCode());
-        Assert.assertEquals(401, result.getErrors().get(0).getAttributes().get("twitter.status.code"));
-        Assert.assertEquals(32, result.getErrors().get(0).getAttributes().get("twitter.error.code"));
+        Assert.assertEquals(VerificationError.StandardCode.AUTHENTICATION, result.getErrors().get(0).getCode());
+        Assert.assertEquals(401, result.getErrors().get(0).getDetails().get("twitter_status_code"));
+        Assert.assertEquals(32, result.getErrors().get(0).getDetails().get("twitter_error_code"));
```

Comment:
Good catch. Will fix that (and will do a grep on `getDeails().get("` to catch potential other spots.